member_database/admin_views.py

```diff
@@ -58,13 +59,14 @@ def inaccessible_callback(self, name, **kwargs):
 class EventView(AuthorizedView):
     access_level = 'event_admin'
     column_list = ['name', 'notify_email', 'max_participants', 'force_tu_mail',
-                   'registration_open']
+                   'registration_open', Event.shortlink]
```

Comment:
Any reason for using the class attribute and not just a link? Better be consistent. If there is an advantage of using the member, let's use it for all columns.

Reply:
No. I thought using the member has advantages, but I don't think it has. (I personally thing this is better anyway, due to autocompletion.)